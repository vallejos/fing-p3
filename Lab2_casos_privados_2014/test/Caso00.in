5
1 -1
2 -1
0 3 -1
4 -1
-1
Filosofia 8 Arte 10 Ciencia 9 -1 -1
Filosofia 4 Ciencia 25 Arte 3 -1 -1
Arte 4 Ciencia 5 -1 -1
Arte 30 Ciencia 25 -1 -1
Ciencia 12 Musica 40 -1 -1
identificarColecciones 3 Filosofia Arte Ciencia
salir
