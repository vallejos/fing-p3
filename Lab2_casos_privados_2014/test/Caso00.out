Tarea 2 - 2014 - Programacion 3.

>0 -> Ciencia
1 -> Arte
2 -> 
>FIN.
